pkg/util/secretutil/secret_test.go

```diff
@@ -597,6 +645,63 @@ g+Ia2YI15BzapW0agqSSTlfGMoQHaPRh1+XYtkOd/xb4xc8d+gc0
 	}
 }
 
+func TestPfxGetPrivateKey(t *testing.T) {
```

Comment:
Instead of a new test, I would recommend adding a new table test in [`TestGetPrivateKey `](https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/pkg/util/secretutil/secret_test.go#L398)

Reply:
I thought about it. But for PFX I am having an input in base64 encoded format and need to decode it before we pass it to the function. It's not needed for existing tests and hence created a new test to keep it clean.

Reply:
You can make actual a function field `func() []byte` and use that as input for `getPrivateKey`. We don't have to worry about the base64 decode error because we pass valid inputs but if we want we can still assert that in the function. Only reason for having all these tests in one is to ensure we always assert the same things for a function unit test.